server/src/instant/storage/s3.clj

```diff
@@ -54,19 +55,25 @@
 (defn signed-upload-url
   ([object-key] (signed-upload-url default-bucket object-key))
   ([bucket-name object-key]
-   (s3/generate-presigned-url {:method :put
-                               :bucket-name bucket-name
-                               :key object-key})))
+   (s3/generate-presigned-url
```

Comment:
Could we make this optional so that people who get this working on OSS won't have to set up the access keys if they don't want to?

Reply:
Done!